src/guides/v2.3/config-guide/varnish/config-varnish-advanced.md

```diff
@@ -82,8 +82,8 @@ bin/magento cache:flush
 
 Saint mode is not part of the main Varnish package. It is a separately-versioned vmod that must be downloaded and installed. As a result, you should re-compile Varnish from source, as described in the following articles:
 
-*  [Installing Varnish 5.1](https://varnish-cache.org/docs/5.1/installation/index.html)
-*  [Installing Varnish 4.1](https://varnish-cache.org/docs/4.1/installation/install.html) (Stable)
+*  [Installing Varnish 6.5](https://varnish-cache.org/docs/6.5/installation/index.html)
+*  [Installing Varnish 5.2](https://varnish-cache.org/docs/5.2/installation/index.html)
```

Comment:
```suggestion
```

5.x is not listed in [Technical requirements](https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements.html#technologies-magento-can-use)

Reply:
As Homer would say "Doh!"